test/fixtures/linter/autofix/ansi/004_indentation/after.sql

```diff
@@ -9,20 +9,23 @@ WITH audience_counts AS (
         list_emails.active != 'D'
     GROUP BY
         user_id,
-        list_id)
+        list_id
+)
 
 SELECT
     user_id,
     list_id,
     audience,
     CASE
         WHEN audience > 0 AND audience <= 200 THEN '< 200'
-        WHEN audience > 200
-             AND audience <= 3000
-             -- NB: This one is a hanging indent, which should pass.
-             AND audience <= 2000 THEN '200 - 2,000'
+        WHEN
+            audience > 200
+            AND audience <= 3000
```

Comment:
Is there a possible configuration which would have done this?
```
            audience > 200 AND audience <= 3000
```

Reply:
Not yet - but let's handle this case as part of "long lines", which I think is the next step. For now, reindent only _adds_ newlines, it never takes them away.

Reply:
Oh cool. Sorry, some of my feedback comes from not following the big picture plan.

I'll try and review the code part of the PR this weekend.